### PR TITLE
Gp 2022.11.12

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 8/08_01_CIEL08_4_1.txt [hunk 1]
```diff
@@ -228,6 +228,7 @@ Hold on, we're meeting here again tomorrow, right?"
 [sha:0087e8a0787a6714fb0dde642bb0eb02a8d1ee09]{
 -- Page 405, Offset 19272.
 -- 「もっちろん！
+// Spruceable?
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_4_1.txt:231]: maybe "Definitely!" would fit the tone better?

File: script/Ciel/Day 8/08_01_CIEL08_5.txt [hunk 2]
```diff
@@ -351,7 +353,8 @@ The scream wrung from the victim of her blow seems to have driven her wild with
 -- Page 445, Offset 19376.
 -- ああもうグズが、モタモタしてんじゃねえよ！　あんな女、外にさえ出ればカンタンに”
 // VA is SCREECHING this one, give that extra a raise
-%{i}Ugh%{/i}, get a fucking move on, you slowpoke!
+// Play around with how to emphasize these nicely
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_5.txt:356]: put my name or something in this if you would so i can grep for the line

File: script/Ciel/Day 8/08_01_CIEL08_5.txt [hunk 3]
```diff
@@ -599,8 +604,9 @@ The nun is taken aback by the vermin's unexpected outburst.
 [sha:769db74da79d9ece1ade7c7618ce836e4a91c222]{
 -- Page 460, Offset 19418.
 -- 彼女は本当に驚いた。獲物がこの期に及んで人間らしい崇高さを見せたから……ではない。
+// Better way to handle double-surprise?
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_5.txt:607]: current version is alright with me

File: script/Ciel/Day 9/09_01_CIEL09_1.txt [hunk 4]
```diff
@@ -442,12 +442,13 @@ Are you not wearing your glasses today?"
...
 -- Page 32, Offset 22217.
 -- 言われて、ペタ、と自分の顔に手を置いた。
-Hearing that, I reach up and touch my face.
+Hearing that, I reach up and check.
 }
 [sha:f6ceec6f7fced6f45096c942c19a5c7af22224a9]{
 -- Page 32, Offset 22218.
 -- そこには裸眼のままの自分がいる。
-I don't have my glasses on.
+// Weirdly bombastic way to phrase it but OK
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_1.txt:450]: yeah this works well

**Review by Avuxo — APPROVED** [review]

File: script/Ciel/Day 8/08_01_CIEL08_5.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_5.txt:356]: // TODO(ben) added

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 5]
```diff
@@ -182,7 +182,7 @@ And just as I was about to act on them, I'd suddenly remember my own name.
 -- Page 513, Offset 19561.
 -- 喉からでそうな悲鳴と泣き出しそうな顔を両手で覆って、その場から逃げ去った。
 When I did, I'd run away as fast as I could, tears welling up in my eyes,
- covering my face and the scream threatening to spill from my throat with both hands.
+ covering my both face and the scream threatening to spill from my throat with my hands.
```
Comment (yk1512) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:185]: both my face

**Review by yk1512 — APPROVED** [review]
🙏

File: script/Ciel/Day 8/08_01_CIEL08_4_1.txt (hunk 1 above)
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_4_1.txt:231]: Sure, why not.

File: script/Ciel/Day 8/08_01_CIEL08_5.txt [hunk 6]
```diff
@@ -582,7 +588,7 @@ The lone survivor takes this opportunity to address a question to her defenseles
 [sha:02c06a17abc3774fdf39863093d050c8ee0208c1]{
 -- Page 459, Offset 19415.
 -- なのにどうして、こんな……助けてやるなんて希望を持たせて殺しやがった!?
-So why... why are you killing us after you gave us false hope we'd be saved?!
+So then why... why would you give us false hope only to kill us?
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_5.txt:591]: I know this was my suggestion and all, but in a way, false hope doesn't quite compute in this construction.
False hope implies it isn't real, so like, giving someone false hope and then killing them is very natural. But he's asking _why_, and if it's false hope, it's obvious why. You wouldn't need to ask; you already implicitly understand.

Though thinking about it this was a problem in the original construction too. Anyway...

>So then why... why would you give us the hope of salvation only to kill us?

File: script/Ciel/Day 8/08_01_CIEL08_5.txt [hunk 7]
```diff
@@ -646,7 +653,7 @@ I'm weak, so I only pick on those even weaker than me.
 // Need to reify the cream -> 甘 sweet/naive a bit for EN
 %{i}Sigh%{/i}, I can't believe it.
  You guys really are rotten to the brain.
- Or is that skull of yours just filled with cream instead?
+ Or is that skull of yours just filled with whipped cream instead?
  It is, isn't it?
  It's because that grey matter of yours is so sweet and naïve that God didn't save you, y'know."
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_5.txt:658]: An alternative metaphor:
Hinged on 甘ったるい meaning broadly more something along the lines of "stupid". So lead = dense = being stupid.

>Or is that lead you have in there?
>It is, isn't it?
>Why else would you be so dense? That's why God didn't save you, y'know.

File: script/Ciel/Day 8/08_01_CIEL08_5.txt [hunk 8]
```diff
@@ -748,7 +755,7 @@ She hasn't even seen them yet,
 [sha:cb90775557fe2793933374bc29b6feb6d4a4f6f6]{
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_5.txt:755]: Above
Just realized he says 牝豚 here, which is probably better as fatass (I think that's more consistent with how I wrote him in Arc route too)

File: script/Ciel/Day 8/08_01_CIEL08_5.txt [hunk 9]
```diff
@@ -835,13 +842,13 @@ I have maintained regular contact, and I'm allowed to act independently, aren't
...
 }
 [sha:62cb8dc79b182a4c9494ae6a0870d05f9a4fcdb6]{
 -- Page 476, Offset 19459.
 -- なんで、近いうちにテメェはオレの預かりだが……スコアも悪い、精神も不安定な不良物件を使うほどマヌケじゃねえ。
 // Again the 'score' thing. Not really worth calling back imo
-So, them's the breaks. You're my problem now, but... your performance sucks, and you've got more than a few screws loose.
+So, them's the breaks. You're %{i}my%{/i} problem now, but... your performance sucks, and you've got more than a few screws loose.
  I'm not dumb enough to use a gun that might go off in my face.
 }
 [sha:12b71e01ce2940729d4f0611369fa38aed159ef2]{
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_5.txt:854]: Below, at least this first mention by Mario of the monastery should be "a monastery"
Offset 19461 is fine though.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 10]
```diff
@@ -320,7 +321,7 @@ I should have had more faith in God.
 // It seems that 息を潜める can have a 'hiding' interpretation, which I've opted for here
 // seeing has how 'waiting with bated breath' reads a lot more eager than I think this actually is?
 During the time I spent locked away in the cage that was my room,
- my heart was slowly ground down until finally it hit a tipping point.
+ my heart was slowly ground down until finally, it hit a tipping point.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:324]: shouldn't it be
>...down, until finally, it hit a tipping point.

alternatively, but I don't think it reads as well.
>...ground down until it finally hit a tipping point.

File: script/Ciel/Day 9/09_01_CIEL09_1.txt [hunk 11]
```diff
@@ -442,12 +442,13 @@ Are you not wearing your glasses today?"
 [sha:39afad2b083b2f183d5c45e93ea69abd963ab5d3]{
 -- Page 32, Offset 22217.
 -- 言われて、ペタ、と自分の顔に手を置いた。
-Hearing that, I reach up and touch my face.
+Hearing that, I reach up and check.
```
Comment (Petrikow) [inline script/Ciel/Day 9/09_01_CIEL09_1.txt:445]: cheek

**Pull request merged**